src/services/reducers/addReducer.js

```diff
@@ -1,17 +1,29 @@
+// import { ADD } from "../constants/constants";
 
-import { ADD } from "../constants/constants";
-
-const initialTodos = { todos: [] }
+const initialTodos = {
+  todos: [
+    { text: "Finish homework", created_at: '27.06.2023', completed: false },
+    { text: "Go grocery shopping", created_at: '27.06.2023', completed: true },
+  ]
+};
 
 const addReducer = (state=initialTodos, action) => {
-  switch (action.type){
-    case ADD:
+  let newTodo;
+  switch (action.type) {
+    case "ADD":
+      newTodo = {
```

Comment:
Avoid declaring variable without using any declaration keyword. Can we avoid initializing task obj here ?